### PR TITLE
699 1370 Fixes for Subscription Product Block view

### PR DESCRIPTION
**Pull request by Georgi2704**
Screenshot 2024-09-10 at 8 52 51
Screenshot 2024-09-10 at 8 52 39

### CODE REVIEW

File: .changeset/slow-jeans-report.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@orchestrator-ui/orchestrator-ui-components': minor
+---
+
+Updated product block view
```
Comment (ricardovdheijden) [inline .changeset/slow-jeans-report.md:5]: Adding the story ID would be nice for traceback in the future

File: packages/orchestrator-ui-components/src/components/WfoSubscription/utils/utils.spec.ts [hunk 2]
```diff
@@ -64,19 +64,6 @@ describe('getProductBlockTitle()', () => {
         );
     });
 
-    it("returns '[title]...'  when title field is to long", () => {
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/components/WfoSubscription/utils/utils.spec.ts:67]: What is the new policy on titles being to long?

**Review by ricardovdheijden — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WfoSubscription/utils/utils.spec.ts (hunk 2 above)
Comment (Georgi2704) [reply packages/orchestrator-ui-components/src/components/WfoSubscription/utils/utils.spec.ts:67]: They are truncated by EuiTextTruncate now. The full title is shown in the product block view.

File: .changeset/slow-jeans-report.md (hunk 1 above)
Comment (Georgi2704) [inline .changeset/slow-jeans-report.md:5]:
```suggestion
699, 1370 - Fix and update product block view
```

**Review by DutchBen — APPROVED** [review]

**Pull request merged**